bundles/org.openhab.binding.icalendar/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.icalendar</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: iCalendar Binding</name>
+
+
+
+  <dependencies>
+    <!-- own dependencies -->
+    <dependency>
+      <groupId>net.sf.biweekly</groupId>
+      <artifactId>biweekly</artifactId>
+      <version>0.6.3</version>
+      <scope>compile</scope>
+      <type>bundle</type>
+    </dependency>
+    <!-- dependencies of biweekly -->
+    <dependency>
+      <groupId>com.github.mangstadt</groupId>
+      <artifactId>vinnie</artifactId>
+      <version>2.0.2</version>
+      <scope>compile</scope>
+      <type>bundle</type>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.10.0.pr3</version>
+      <scope>compile</scope>
+      <type>bundle</type>
+    </dependency>
```

Comment:
Yes, jersey-all also exports a jackson. But it's version 2.4.5 and biweekly requires 2.9 minimum. That's why it's added here in this way.

Reply:
Jackson's overall api rarely ever changes (at least not from my experience with it). Is it possible that biweekly can work with an older version of jackson?

Reply:
This can only be possible if we integrate sources directly into the bundle (but i have not reviewed biweekly in that detail, that i know why it requires exactly 2.9 or newer). biweekly is a proper OSGi-Bundle and defines the dependencies to jackson as version "[2.9,3)".

I think using a second version of jackson is in this case the cleaner way.